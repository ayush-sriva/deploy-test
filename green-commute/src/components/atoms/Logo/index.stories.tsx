import Logo from '.';
import { ComponentMeta, ComponentStory } from '@storybook/react';

const LogoComponent = {
  title: 'Components/Atoms/Logo',
  component: Logo,
} as ComponentMeta<typeof Logo>;

export const Template: ComponentStory<typeof Logo> = (args: any) => <Logo {...args} />;

Template.args = {
  url: '/assets/logos/logoGreen.png',
};

export default LogoComponent;
