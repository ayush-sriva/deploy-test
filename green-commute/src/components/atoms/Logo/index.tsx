/**
 * @author Asish Kalintha <asish.sugun@zemosolabs.com>
 */

import React from 'react';
import { CommonProps } from '@mui/material/OverridableComponent';

interface LogoProps {
  url: string;
}

const Logo:React.FC<LogoProps & CommonProps> = ({ url } ) =>  <img data-testid='logo' src={url} alt='Logo'/>;


export default Logo;
