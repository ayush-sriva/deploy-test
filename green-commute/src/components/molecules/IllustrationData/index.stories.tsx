import IllustrationData from '.';
import { ComponentMeta, ComponentStory } from '@storybook/react';

export default {
  title: 'Components/Molecules/IllustrationData',
  component: IllustrationData,
} as ComponentMeta<typeof IllustrationData>;

const IllustrationDataTemplate: ComponentStory<typeof IllustrationData> = (args: any) => (
  <div style={{ background: 'linear-gradient(155.94deg, #EFFEFF 6.2%, #E9FFF4 52.61%)', padding: '30px' }}>
    {' '}
    <IllustrationData {...args} />
  </div>
);

export const IllustrationData1 = IllustrationDataTemplate.bind({});
IllustrationData1.args = {
  url: '/assets/illustrations/illustration2.png',
  children: 'Enter Location to know Time Air Quality index (AQI)',
};

export const IllustrationData2 = IllustrationDataTemplate.bind({});
IllustrationData2.args = {
  url: '/assets/illustrations/illustration4.png',
  children: 'Enter Location to know Time Air Quality (AQI)',
};

export const IllustrationData3 = IllustrationDataTemplate.bind({});
IllustrationData3.args = {
  url: '/assets/illustrations/illustration3.png',
  children: 'Enter your Skills to know how many jobs are in this Location',
};
