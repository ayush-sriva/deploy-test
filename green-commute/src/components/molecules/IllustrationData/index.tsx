/**
 * @author Asish Kalintha <asish.sugun@zemosolabs.com>
 */
import { Grid } from '@mui/material';
import { makeStyles } from '@mui/styles';
import Typography from '../../atoms/Typography/index';
import Illustration from '../../atoms/Illustration';
import responsiveTheme from '../../../theme';
import { ReactNode } from 'react';

interface IProps {
  children: ReactNode;
  url: string;
}

const useStyles = makeStyles({
  container: {
    maxWidth: '417px',
    height: '348px',
    justifyContent: 'center',
    alignItems: 'center',
  },
  text: {
    color: `${responsiveTheme.palette.secondary.dark}`,
    textAlign: 'center',
    marginTop: '40px',
  },
});

const IllustrationData:React.FC<IProps> = ({url, children}) => {
    const classes = useStyles();
  return (
    <div data-testid='illustrationData'>
      <Grid
        container
        direction='column'
        className={classes.container}
        wrap='nowrap'
       
      >
        <Grid item>
          <Illustration url={url} />
        </Grid>

        <Grid item>
          <Typography
            variant='h2'
            className={classes.text}
            
          >
            {children}
          </Typography>
        </Grid>
      </Grid>
    </div>
  );
};

export default IllustrationData;
