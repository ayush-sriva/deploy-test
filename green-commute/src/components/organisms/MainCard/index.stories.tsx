import { ComponentStory, ComponentMeta } from '@storybook/react';
import MainCard from '.';

export default {
  title: 'components/organisms/Main Card',
  component: MainCard,
} as ComponentMeta<typeof MainCard>;

const Template: ComponentStory<typeof MainCard> = (args) => <MainCard {...args} />;

export const DemoMainCard = Template.bind({});

DemoMainCard.args = {};
