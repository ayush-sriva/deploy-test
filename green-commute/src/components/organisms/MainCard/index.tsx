import theme from '../../../theme';
import { Container } from '@mui/material';
import Grid from '@mui/material/Grid';
import Button from '../../atoms/Button';
import InputField from '../../atoms/InputField';
import SingleInputField from '../../atoms/TextField';
import TypographyComponent from '../../atoms/Typography';
import TabPanel from '../TabPanel';
import React, { useContext } from 'react';
import { CommonProps } from '@mui/material/OverridableComponent';
import { styled } from '@mui/material';
import { JobLocationContext, UserLocationContext, UserSkillsContext } from '../../pages/LandingPage';

interface TabPanelProps {
  searchBarText?: string;
  currentPage: number;
  backClickHandlerWrap: React.MouseEventHandler<HTMLButtonElement>;
  nextClickHandlerWrap: React.MouseEventHandler<HTMLButtonElement>;
}

const ButtonGrid = styled(Grid)({
  marginTop: '40px !important',
});

const GridTab = styled(Grid)({
  height: '60px',
});

const Skip = styled(TypographyComponent)({
  width: '38px',
  marginTop: '60px !important',
  color: `${theme.palette.primary.main}`,
  borderBottom: `2px solid ${theme.palette.primary.main}`,
});

const Subtitle = styled(TypographyComponent)({
  marginBottom: '12px !important',
});

const SpacedButton = styled('div')({
  display: 'inline',
  padding: '0px',
  paddingRight: '12px',
});

const MainContainer = styled(Container)({
  width: '672px',
  marginLeft: '0px',
  paddingLeft: '0px',
});

const TypoGrid = styled(Grid)({
  marginTop: '72px',
  marginBottom: '64px',
});

const MainCard: React.FC<TabPanelProps & CommonProps> = ({
  currentPage,
  backClickHandlerWrap,
  nextClickHandlerWrap,
}) => {
  const locationContext = useContext(UserLocationContext);
  let jobContext = useContext(JobLocationContext);
  let skillContext = useContext(UserSkillsContext);
  let jobLocationOptions = ['Hyderabad', 'Mumbai', 'Delhi', 'Bangalore', 'Chennai'];
  let userSkillsOptions = ['UI/UX Designer', 'Graphic Designer', 'Python Developer'];

  //for just checking the feature, implementation will be added in pages
  const handleClick = () => {
    console.info('You clicked the Chip.');
  };

  //for just checking the feature, implementation will be added in pages
  const handleDelete = () => {
    console.info('You clicked the delete icon.');
  };

  const wrapLocationHandler = (location: string[]) => {
    jobContext.setJobLocation(location);
    console.log('current location: ' + location);
  };

  const wrapSkillHandler = (skill: string[]) => {
    console.log('skill data ' + skill);
    skillContext.setUserSkills(skill);
  };

  const wrapUserLocationHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
    locationContext.setUserLocation(event.target.value);
    console.log(locationContext.userLocation);
  };

  const renderInputField = (currentPage: number) => {
    if (currentPage === 0) {
      return (
        <SingleInputField
          placeholderText={'Enter your location'}
          value={locationContext.userLocation}
          eventHandler={wrapUserLocationHandler}
        />
      );
    } else if (currentPage === 1) {
      return (
        <InputField
          key={'current input'}
          placeholder={'Enter your job location'}
          handleClick={handleClick}
          handleDelete={handleDelete}
          eventHandler={wrapLocationHandler}
          values={jobContext.jobLocation}
          options={jobLocationOptions}
        ></InputField>
      );
    } else if (currentPage === 2) {
      return (
        <InputField
          key={'new input'}
          placeholder={'Enter your skills'}
          handleClick={handleClick}
          handleDelete={handleDelete}
          eventHandler={wrapSkillHandler}
          values={skillContext.userSkills}
          options={userSkillsOptions}
        ></InputField>
      );
    }
  };

  const renderSearchBarText = (currentPage: number) => {
    if (currentPage === 0) {
      return <Subtitle variant='subtitle1'>Where do you stay?</Subtitle>;
    } else if (currentPage === 1) {
      return <Subtitle variant='subtitle1'>Where do you want to work?</Subtitle>;
    } else if (currentPage === 2) {
      return <Subtitle variant='subtitle1'>What do you want to do?</Subtitle>;
    }
  };

  return (
    <MainContainer>
      <Grid container direction='column' justifyContent='center'>
        <GridTab item>{currentPage === 0 ? <TabPanel></TabPanel> : <TabPanel pageId={currentPage} />}</GridTab>
        <TypoGrid item>
          <TypographyComponent variant='h1'>More than 2000 people are using Green Commute</TypographyComponent>
        </TypoGrid>
        <Grid item>
          {renderSearchBarText(currentPage)}
          {renderInputField(currentPage)}
        </Grid>
        <ButtonGrid item>
          {currentPage > 0 ? (
            <SpacedButton>
              <Button
                color='primary'
                buttonVariant='outlined'
                type='reset'
                typoVariant='body1'
                onClickHandler={backClickHandlerWrap}
                size='large'
              >
                Back
              </Button>
            </SpacedButton>
          ) : (
            ''
          )}
          <Button
            color='primary'
            buttonVariant='contained'
            type='button'
            typoVariant='body1'
            onClickHandler={nextClickHandlerWrap}
            size='medium'
          >
            Next
          </Button>
        </ButtonGrid>
        <Grid item>
          <Skip variant='subtitle1'>Skip</Skip>
        </Grid>
      </Grid>
    </MainContainer>
  );
};

MainCard.defaultProps = {
  searchBarText: 'Where do you stay?',
};

export default MainCard;
