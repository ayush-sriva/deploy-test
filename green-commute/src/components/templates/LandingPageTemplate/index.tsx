import { Box, Grid, styled } from '@mui/material';
import responsiveTheme from '../../../theme';

interface LandingPageProps {
  children: React.ReactNode;
  logoChildren: React.ReactNode;
  cardChildren: React.ReactNode;
}

const StyledLogo = styled(Grid)({
  marginTop: '2rem',
  marginBottom: '4rem',
  paddingLeft: '16px',
});

const IllustrationGrid = styled(Grid)({
  justifyContent: 'center',
  alignItems: 'center',
  background: `linear-gradient(155.94deg, ${responsiveTheme.palette.success.main} 6.2%, ${responsiveTheme.palette.success.dark} 52.61%)`,
  height: '98vh',
  paddingBottom: '80px',
});

const RightContainer = styled(Grid)({
  padding: '0px',
});

const StyledBox = styled(Box)({
  width: '1366px',
  margin: 'auto',
  position: 'relative',
});

const LeftContainer = styled(Grid)({
  paddingLeft: '100px',
});

const LandingPageTemplate: React.FC<LandingPageProps> = ({ children, logoChildren, cardChildren }) => {
  return (
    <StyledBox>
      <Grid container display={'flex'} direction='row' flexWrap={'nowrap'}>
        <LeftContainer item xs={7}>
          <Grid container>
            <StyledLogo item xs={12}>
              {logoChildren}
            </StyledLogo>
            <Grid item xs={12}>
              {cardChildren}
            </Grid>
          </Grid>
        </LeftContainer>
        <RightContainer item xs={5}>
          <IllustrationGrid container>{children}</IllustrationGrid>
        </RightContainer>
      </Grid>
    </StyledBox>
  );
};

export default LandingPageTemplate;
