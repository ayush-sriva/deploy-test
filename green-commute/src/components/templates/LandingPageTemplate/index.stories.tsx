import { ComponentStory, ComponentMeta } from '@storybook/react';
import React from 'react';
import Logo from '../../atoms/Logo';
import MainCard from '../../organisms/MainCard';
import LandingPageTemplate from './index';
import useState from 'storybook-addon-state';

export default {
  title: 'Components/Templates/LandingPage',
  component: LandingPageTemplate,
} as ComponentMeta<typeof LandingPageTemplate>;

const [currentPage, setCurrentPage] = useState<number>('Current page', 0);

const backClickHandlerWrap = (_event: React.MouseEvent<HTMLButtonElement>) => {
  if (currentPage > 0) {
    setCurrentPage(currentPage - 1);
  }
};

const nextClickHandlerWrap = (_event: React.MouseEvent<HTMLButtonElement>) => {
  if (currentPage < 2) {
    setCurrentPage(currentPage + 1);
  }
};

const Template: ComponentStory<typeof LandingPageTemplate> = (args) => <LandingPageTemplate {...args}/>;

export const PrimaryTemplate = Template.bind({});

PrimaryTemplate.args = {
    children: '',
    logoChildren: <Logo url='assets/logos/logoGreen.png' />,
    cardChildren: <MainCard currentPage={currentPage} backClickHandlerWrap={backClickHandlerWrap} nextClickHandlerWrap={nextClickHandlerWrap}/>
}

