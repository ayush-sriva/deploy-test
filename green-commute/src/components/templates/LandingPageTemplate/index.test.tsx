import { render, screen } from '@testing-library/react';
import React, { useState } from 'react';
import renderer from 'react-test-renderer';
import LandingPageTemplate from '.';
import Logo from '../../atoms/Logo';
import IllustrationData from '../../molecules/IllustrationData';
import MainCard from '../../organisms/MainCard';

const renderLandingPage = () => {
  let currentPage = 0;

  const backClickHandlerWrap = (_event: React.MouseEvent<HTMLButtonElement>) => {
    if (currentPage > 0) {
      currentPage = currentPage - 1;
    }
  };

  const nextClickHandlerWrap = (_event: React.MouseEvent<HTMLButtonElement>) => {
    if (currentPage < 2) {
      currentPage = currentPage + 1;
    }
  };

    render(<LandingPageTemplate children={<IllustrationData url= '/assets/illustrations/illustration2.png' children= {'Enter Location to know Time Air Quality Index(AQI)'}/>} logoChildren={ <Logo url='assets/logos/logoGreen.png' />} cardChildren={<MainCard currentPage={currentPage} backClickHandlerWrap={backClickHandlerWrap} nextClickHandlerWrap={nextClickHandlerWrap}/>}/>)
}
describe('LandingPageTemplate', () => {
    it('LandingPageTemplate snapshot', () => {
      let currentPage = 0;

  const backClickHandlerWrap = (_event: React.MouseEvent<HTMLButtonElement>) => {
    if (currentPage > 0) {
      currentPage = currentPage - 1;
    }
  };

  const nextClickHandlerWrap = (_event: React.MouseEvent<HTMLButtonElement>) => {
    if (currentPage < 2) {
      currentPage = currentPage + 1;
    }
  };  

        const tree = renderer
        .create(<LandingPageTemplate children={<IllustrationData url= '/assets/illustrations/illustration2.png' children= {'Enter Location to know Time Air Quality Index(AQI)'}/>} logoChildren={ <Logo url='assets/logos/logoGreen.png' />} cardChildren={<MainCard currentPage={currentPage} backClickHandlerWrap={backClickHandlerWrap} nextClickHandlerWrap={nextClickHandlerWrap}/>}/>)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });

    test('Renders logo correctly', () => {
        renderLandingPage();
        const logoElement = screen.getByTestId('logo');
        expect(logoElement).toBeInTheDocument();
    });

    test('Renders illustration correctly', () => {
        renderLandingPage();
        const illustrationElement = screen.getByAltText('illustration');
        expect(illustrationElement).toBeInTheDocument();
    });

    test('Renders text correctly', () => {
        renderLandingPage();
        let textElement = screen.getByText("More than 2000 people are using Green Commute");
        expect(textElement).toBeInTheDocument();
        textElement = screen.getByText("Enter Location to know Time Air Quality Index(AQI)");
        expect(textElement).toBeInTheDocument();
    });

    test('Renders input correctly', () => {
        renderLandingPage();
        const placeholderText = screen.getByPlaceholderText('Enter your location');
        expect(placeholderText).toBeInTheDocument();
    })
})