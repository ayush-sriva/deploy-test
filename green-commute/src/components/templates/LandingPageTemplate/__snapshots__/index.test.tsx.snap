// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPageTemplate LandingPageTemplate snapshot 1`] = `
<div
  className="MuiBox-root css-1pspca3"
>
  <div
    className="MuiGrid-root MuiGrid-container css-1eqh52y-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7 css-61hfwp-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-y3ngzy-MuiGrid-root"
        >
          <img
            alt="Logo"
            data-testid="logo"
            src="assets/logos/logoGreen.png"
          />
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
        >
          <div
            className="MuiContainer-root MuiContainer-maxWidthLg css-1phxsb3-MuiContainer-root"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-6gh8l0-MuiGrid-root"
            >
              <div
                className="MuiGrid-root MuiGrid-item css-8q5yis-MuiGrid-root"
              >
                <div
                  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
                >
                  <div
                    className="MuiGrid-root MuiGrid-item css-1p1lgg-MuiGrid-root"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
                      data-testid="item-container"
                    >
                      <div
                        className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                      >
                        <div
                          className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
                        >
                          1
                        </div>
                      </div>
                      <div
                        className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                      >
                        <h6
                          className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignJustify css-669lo9-MuiTypography-root"
                        >
                          Your Location
                        </h6>
                      </div>
                    </div>
                  </div>
                  <div
                    className="MuiGrid-root MuiGrid-item css-1p1lgg-MuiGrid-root"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
                      data-testid="item-container"
                    >
                      <div
                        className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                      >
                        <div
                          className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
                        >
                          2
                        </div>
                      </div>
                      <div
                        className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                      >
                        <h6
                          className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignJustify css-tv3ey-MuiTypography-root"
                        >
                          Job Location
                        </h6>
                      </div>
                    </div>
                  </div>
                  <div
                    className="MuiGrid-root MuiGrid-item css-1p1lgg-MuiGrid-root"
                  >
                    <div
                      className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
                      data-testid="item-container"
                    >
                      <div
                        className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                      >
                        <div
                          className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
                        >
                          3
                        </div>
                      </div>
                      <div
                        className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
                      >
                        <h6
                          className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-alignJustify css-tv3ey-MuiTypography-root"
                        >
                          Your Skills
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item css-1df6ge6-MuiGrid-root"
              >
                <h1
                  className="MuiTypography-root MuiTypography-h1 css-o2w69a-MuiTypography-root"
                >
                  More than 2000 people are using Green Commute
                </h1>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              >
                <h6
                  className="MuiTypography-root MuiTypography-subtitle1 css-1mdleez-MuiTypography-root"
                >
                  Where do you stay?
                </h6>
                <div
                  className="MuiFormControl-root MuiTextField-root css-omd3jb-MuiFormControl-root-MuiTextField-root"
                >
                  <div
                    className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                    onClick={[Function]}
                  >
                    <input
                      aria-invalid={false}
                      autoFocus={false}
                      className="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                      disabled={false}
                      id=":r0:"
                      onAnimationStart={[Function]}
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      placeholder="Enter your location"
                      required={false}
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden={true}
                      className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        className="css-hdw1oc"
                      >
                        <span
                          className="notranslate"
                        >
                          â€‹
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item css-1h8ojxo-MuiGrid-root"
              >
                <button
                  className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root"
                  disabled={false}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={0}
                  type="button"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    Next
                  </p>
                </button>
              </div>
              <div
                className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
              >
                <h6
                  className="MuiTypography-root MuiTypography-subtitle1 css-1223bt7-MuiTypography-root"
                >
                  Skip
                </h6>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-1kwfdhe-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-container css-57a9o0-MuiGrid-root"
      >
        <div
          data-testid="illustrationData"
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-wrap-xs-nowrap makeStyles-container-1 css-19kb62l-MuiGrid-root"
          >
            <div
              className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <img
                alt="illustration"
                src="/assets/illustrations/illustration2.png"
              />
            </div>
            <div
              className="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
            >
              <h2
                className="MuiTypography-root MuiTypography-h2 makeStyles-text-2 css-1sra7t5-MuiTypography-root"
              >
                Enter Location to know Time Air Quality Index(AQI)
              </h2>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
